app/ui.js

```diff
@@ -502,7 +502,6 @@ var UI;
             }
 
             UI.enableDisableViewClip();
-            document.getElementById('noVNC_setting_resize').disabled = UI.connected;
```

Comment:
It doesn't here. Changing the setting doesn't do anything. I have to resize the window for things to take effect. Also, switching from "scaling" to "none" is just broken.